services/indexes/pvm/writer.go

```diff
@@ -397,29 +405,49 @@ func (w *Writer) indexBlock(ctx services.ConsumerCtx, proposerblockBytes []byte)
 	errs := wrappers.Errs{}
 
 	switch blk := pblock.(type) {
-	case *platformvm.ProposalBlock:
+	case *blocks.ApricotProposalBlock:
 		errs.Add(
 			initializeTx(ver, w.codec, blk.Tx),
 			w.indexCommonBlock(ctx, blkID, models.BlockTypeProposal, blk.CommonBlock, blockBytes),
-			w.indexTransaction(ctx, blkID, blk.Tx, false),
+			w.indexTransaction(ctx, blkID, *blk.Tx, false),
 		)
-	case *platformvm.StandardBlock:
+	case *blocks.ApricotStandardBlock:
 		errs.Add(w.indexCommonBlock(ctx, blkID, models.BlockTypeStandard, blk.CommonBlock, blockBytes))
-		for _, tx := range blk.Txs {
+		for _, tx := range blk.Transactions {
 			errs.Add(
-				initializeTx(ver, w.codec, *tx),
+				initializeTx(ver, w.codec, tx),
 				w.indexTransaction(ctx, blkID, *tx, false),
 			)
 		}
-	case *platformvm.AtomicBlock:
+	case *blocks.ApricotAtomicBlock:
 		errs.Add(
 			initializeTx(ver, w.codec, blk.Tx),
 			w.indexCommonBlock(ctx, blkID, models.BlockTypeProposal, blk.CommonBlock, blockBytes),
-			w.indexTransaction(ctx, blkID, blk.Tx, false),
+			w.indexTransaction(ctx, blkID, *blk.Tx, false),
 		)
-	case *platformvm.AbortBlock:
+	case *blocks.ApricotAbortBlock:
+		errs.Add(w.indexCommonBlock(ctx, blkID, models.BlockTypeAbort, blk.CommonBlock, blockBytes))
+	case *blocks.ApricotCommitBlock:
+		errs.Add(w.indexCommonBlock(ctx, blkID, models.BlockTypeCommit, blk.CommonBlock, blockBytes))
+	case *blocks.BlueberryProposalBlock:
```

Comment:
Can probably condense blueberryProposal/Standard blocks into a single case.  Leaving alone for now for clarity